mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/TopicFilter.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamnative.pulsar.handlers.mqtt;
+
+public interface TopicFilter {
+
+    TopicFilterDisabled DISABLED = new TopicFilterDisabled();
+
+    String SINGLE_LEVEL = "+";
+    String MULTI_LEVEL = "#";
+    String SPLITTER = "/";
+
+    boolean test(String localTopicName);
+
+    class TopicFilterDisabled implements TopicFilter {
+
+        @Override
+        public boolean test(String localTopicName) {
+            return true;
```

Comment:
If users use the `TopicFilterDisabled`, the filter will match all topics successfully, maybe it's a potential risk?

Reply:
Oh, the DISABLED is not useful for now, will remove it.